### DISCUSSION TITLE
ARM64 tests need updating

### DISCUSSION THREAD

**Discussion by pcarruscag** [su2code/SU2#1776] [General]
I'll take care of it soon.

**Comment by martin-g** [comment]
There were some tests which produced different values on every run: https://github.com/su2code/SU2/pull/1619#issuecomment-1164226451
I was not able to find a way to make them stable...

**Reply by pcarruscag** [reply]
Polar Naca 0012 right? I did not investigate it.

**Reply by martin-g** [reply]
Yes! It was failing in two **Regression.py suites.
